devito/profiler.py

```diff
@@ -81,18 +86,23 @@ def add_profiling(self, code, name, byte_size=4, omp_flag=None):
                           Defaults to 4.
         :param omp_flag: OpenMP flag to add before profiling operations,
                          if needed.
+        :param to_ignore: List of strings containing the labels of
+                          symbols used as loop variables
```

Comment:
Sorry: I was talking about loop indices, not actual ``Iteration`` objets (I confused the names), as in the docstring of ``to_ignore`` I see "List of strings containing the labels of symbols used as loop variables"